src/content/learn/managing-state.md

```diff
@@ -4,27 +4,27 @@ title: Managing State
 
 <Intro>
 
-As your application grows, it helps to be more intentional about how your state is organized and how the data flows between your components. Redundant or duplicate state is a common source of bugs. In this chapter, you'll learn how to structure your state well, how to keep your state update logic maintainable, and how to share state between distant components.
+Ketika aplikasi Anda berkembang, penting untuk lebih memperhatikan bagaimana keadaan (*state*) Anda diorganisasi dan bagaimana aliran data antara komponen-komponen Anda. Keadaan yang redundan atau duplikat adalah sumber umum dari bug. Di bab ini, Anda akan belajar bagaimana mengatur keadaan (*state*) dengan baik, bagaimana menjaga logika pembaruan keadaan (*state*) agar mudah dikelola, dan bagaimana berbagi keadaan (*state*) antara komponen-komponen yang jauh.
```

Comment:
```suggestion
Ketika aplikasi Anda berkembang, penting untuk lebih memperhatikan bagaimana *state* Anda diorganisasi dan bagaimana aliran data antara komponen-komponen Anda. Keadaan yang redundan atau duplikat adalah sumber umum dari bug. Di bab ini, Anda akan belajar bagaimana mengatur *state* dengan baik, bagaimana menjaga logika pembaruan *state* agar mudah dikelola, dan bagaimana berbagi *state* antara komponen-komponen yang jauh.
```

According to the [glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium#s), we should NOT translate `state` to `keadaan`. The reason being `state` is a literal syntax actively used in React.
Therefore, please update all the "keadaan (*state*)" occurrences down below into "*state*". Thanks!